main.cpp

```diff
@@ -1,23 +1,197 @@
 #include <iostream>
 #include <string>
 #include <string_view>
+#include <sstream>
+#include <vector>
 #include <algorithm>
 
 namespace
 {
+int random_no(unsigned int range)
+{
+	srand(time(NULL));
+	int secret_num = rand() % range;
+	return secret_num;
+}
+
 std::string to_lowercase(std::string_view word)
 {
 	std::string lowercased = word.begin();
 	std::transform(word.begin(), word.end(), lowercased.begin(), tolower);
 	return lowercased;
 }
+
+void initialize_combolist(std::stringstream& combo_names, std::vector<std::string>& combo_list)
+{
+	std::string combo_name;
+	while (getline(combo_names, combo_name, ' '))
+	{
+		combo_list.push_back(combo_name);
+	}
+}
+
+void display_rules()
+{
+	std::cout << "> RULES: \n";
+	std::cout << "> 1. If a number is guessed number of tries goes down by 1\n";
+	std::cout << "> 2. If a character that is guessed is not in the word \n";
+	std::cout << "     the number of tries goes down by 1\n\n";
+}
+
+void start_screen()
+{
+	std::cout << "\n== HANGMAN ==\n";
+	std::cout << "> Press 0 to quit at any time\n\n";
+	std::cout << "> OBJECTIVE: \n";
+	std::cout << "> Guess the word before running out of tries\n\n";
+	display_rules();
+}
+
+void ask_for_input(char* dest)
+{
+	std::cout << "> ";
+	std::cin >> *dest;
```

Comment:
If the player enters more than one character, this approach will probably not work the way they would expect it to.